src/homepage/pages/Docs.vue

```diff
@@ -38,7 +38,7 @@
     }
   };
 </script>
-<style lang="scss" scope>
+<style lang="scss" rel="stylesheet/scss">
```

Comment:
Sorry, I will put it back. And according to https://vue-loader.vuejs.org/en/features/scoped-css.html, `scope` should be `scoped`.